jbpm-wb-human-tasks/jbpm-wb-human-tasks-api/src/main/java/org/jbpm/workbench/ht/model/events/TaskSelectionEvent.java

```diff
@@ -61,6 +70,62 @@ public TaskSelectionEvent(String serverTemplateId,
         this.forLog = forLog;
     }
 
+    public TaskSelectionEvent(String serverTemplateId,
```

Comment:
Could you add the related methods `equals`, `hascode` and `toString`? Another suggestion would be to move `forAdmin` and `forLog` to Boolean instead.